.github/fabricbot.json

```diff
@@ -1,939 +0,0 @@
-                  }
-                },
-                {
-                  "name": "titleContains",
-                  "parameters": {
-                    "titlePattern": "[Proposal]"
-                  }
-                }
-              ]
-            },
-            {
-              "operator": "not",
-              "operands": [
-                {
-                  "name": "isOpen",
-                  "parameters": {}
-                }
-              ]
-            },
-            {
-              "operator": "not",
-              "operands": [
-                {
-                  "operator": "or",
-                  "operands": [
-                    {
-                      "name": "isInProjectColumn",
-                      "parameters": {
-                        "projectName": "New Feature Proposals",
-                        "columnName": "Closed"
-                      }
-                    },
-                    {
-                      "name": "isInProjectColumn",
-                      "parameters": {
-                        "projectName": "New Feature Proposals",
-                        "columnName": "Completed"
-                      }
-                    }
-                  ]
-                }
-              ]
-            }
-          ]
-        },
-        "eventType": "issue",
-        "eventNames": [
-          "issues",
-          "project_card"
-        ],
-        "taskName": "Reopen Closed Proposals Not In Closed Column",
-        "actions": [
-          {
-            "name": "reopenIssue",
-            "parameters": {}
-          },
-          {
-            "name": "addReply",
-            "parameters": {
-              "comment": "Reopening Proposal. \n\nOnly Proposals moved to the [`Closed` Project Column](https://github.com/CommunityToolkit/Maui/projects/1) and [`Completed` Project Column](https://github.com/CommunityToolkit/Maui/projects/1) can be closed."
-            }
-          }
-        ]
-      }
-    },
-    {
-      "taskType": "trigger",
-      "capabilityId": "IssueResponder",
-      "subCapability": "IssuesOnlyResponder",
-      "version": "1.0",
-      "id": "HzB1QRBDGmPd65D-yROFt",
-      "config": {
-        "conditions": {
-          "operator": "or",
-          "operands": [
-            {
-              "name": "addedToProjectColumn",
-              "parameters": {
-                "projectName": "New Feature Proposals",
-                "columnName": "Completed"
-              }
-            },
-            {
-              "name": "addedToProjectColumn",
-              "parameters": {
-                "projectName": "New Feature Proposals",
-                "columnName": "Closed"
-              }
-            }
-          ]
-        },
-        "eventType": "issue",
-        "eventNames": [
-          "issues",
-          "project_card"
-        ],
-        "taskName": "Automatically Close Proposals when Added to `Completed` or `Closed`",
```

Comment:
Project board automation is not supported in the new service, so related tasks were not ported.